sig-multicluster/terminology.md

```diff
@@ -0,0 +1,39 @@
+# Terminology
+
+This file contains terminology in common use by the SIG-Multicluster community, as well as aliases those terms may be used by past or present.
+
+For more information, see individual subprojects and documents, or the SIG-Multicluster website at https://multicluster.sigs.k8s.io/.
+
+## About API
+
+### Also known as:
+
+- ClusterProperty
+
+### Previously known as:
+
+-  ClusterClaim API
+
+
+## ClusterProfile API
+
+### Previously known as:
+
+- Cluster Inventory
```

Comment:
But this term is now used as the repo name and in the readme https://github.com/kubernetes-sigs/cluster-inventory-api.

Reply:
Yea I had though that while this is no longer the name of the CRD it was actually intended to preserve as the name of the API? (Similar to About API <> ClusterProperty)

Defer to @ryanzhang-oss @corentone for a more authoritative answer.

Reply:
This one is a bit different to About API <> ClusterProperty to me because the Kind is also called ClusterProfile. That being said if the common usage is to refer to it as Cluster Inventory regardless of the API and Kind name then this PR should reflect that. I think this one got in its state at least partially because the _repo_ name is Cluster Inventory API and that is harder to change from the Github beauracracy perspective.... anyways part of this effort could be that we decide to cleanup our documents _IF_ we think Cluster Inventory shouldn't be a colloquial name for this.

But overall ya I defer to @ryanzhang-oss and @corentone or their delegates and will update accordingly :)

Reply:
For now I removed it and we can discuss this on a follow up PR